ci-operator/config/calebevans/windup_integration_test/calebevans-windup_integration_test-mtr.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  to: mtr-runner
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.12"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtr-scenario
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      SELENIUM_NAMESPACE: mtr
+      SUB_INSTALL_NAMESPACE: mtr
+      SUB_PACKAGE: mtr-operator
```

Comment:
Is there a way we can specify MTR version here? Is there a way you've been able to tell which version is being installed? From the artifacts I don't see any version data. [artifacts](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/34094/rehearse-34094-pull-ci-calebevans-windup_integration_test-mtr-mtr-scenario/1603892577847742464/artifacts/mtr-scenario/interop-tooling-operator-install/)

I believe since the update to MTR the operator is just at 1.0 but this will become important when we eventually do our expansion to 1.1

Here's an example of specifying the version for the ACM operator
[example](https://github.com/openshift/release/pull/34519/files#diff-e31942882c118aea60054d7a94d29110fd0eb96c45b748bd2cab50c9b49d87f9R38)

Though this method isn't ideal as it will make expansions a manual process ideally we can support multiple versions and support expansions automatically, this needs some brainstorming.

Reply:
Just putting this here for posterity, as we spoke about this in the Slack channel:

We can add another test that specifies which version of the operator to install, that way all required versions are tested:

```yaml
- as: mtr-scenario-1.0.0
  steps:
    cluster_profile: aws-cspi-qe
    env:
      BASE_DOMAIN: aws.interop.ccitredhat.com
      SELENIUM_NAMESPACE: mtr
      SUB_INSTALL_NAMESPACE: mtr
      SUB_PACKAGE: mtr-operator
      SUB_SOURCE: redhat-operators
      SUB_TARGET_NAMESPACES: mtr
      SUB_CHANNEL: stable-v1.0
    test:
    - chain: interop-mtr
    workflow: cucushift-installer-rehearse-aws-ipi
- as: mtr-scenario-2.0.0
  steps:
    cluster_profile: aws-cspi-qe
    env:
      BASE_DOMAIN: aws.interop.ccitredhat.com
      SELENIUM_NAMESPACE: mtr
      SUB_INSTALL_NAMESPACE: mtr
      SUB_PACKAGE: mtr-operator
      SUB_SOURCE: redhat-operators
      SUB_TARGET_NAMESPACES: mtr
      SUB_CHANNEL: stable-v2.0
    test:
    - chain: interop-mtr
    workflow: cucushift-installer-rehearse-aws-ipi
```